openassessment/data.py

```diff
@@ -993,15 +999,37 @@ def create_zip_with_attachments(cls, file, course_id, submission_files_data):
 
         with ZipFile(file, 'w') as zip_file:
```

Comment:
Update docstring to note that missing files will not be included in the archive.